package.json

```diff
@@ -122,6 +122,8 @@
   },
   "dependencies": {
     "@babel/polyfill": "^7.12.1",
+    "@jamescoyle/vue-icon": "^0.1.2",
+    "@mdi/js": "^6.1.95",
```

Comment:
Do we really need this packages?

Reply:
As I understand it, the webpack tree shaking should only include the few bytes for each used icon - that's why I'm using the recommended way for Vue and don't use the web font (which is huge).
It may take even less if existing icons would be used that way as well ...

Reply:
> Hi @ahochsteger , thanks for your PR! I love to keep the code as simple as possible, if it was me I would have choose to set the battery level as a node property instead. It is also used on mesh graph so it makes sense to have it there. Just remember to add the code to the value update in order to keep it updated

@robertsLando, thanks for your feedback, that confirms my thoughts while implementing this.
Is there already an implementation of a similar property that syncs the value property I could use as reference? Maybe you have a pointer for me ...

Reply:
> Is there already an implementation of a similar property that syncs the value property I could use as reference? Maybe you have a pointer for me ...

Unfortunally nope, but I made my comment before to see your actual implementation and I think it looks good anyway and could be helpful in future for other addition on the table

Reply:
About the icons if it doesn't add the entire icons font I'm ok for them :)

Reply:
> Unfortunally nope, but I made my comment before to see your actual implementation and I think it looks good anyway and could be helpful in future for other addition on the table

Ok, so I'll fix the sorting and grouping issue for the table and keep accessing the value using the `valueFn` for now. We have to keep an eye on the performance though, since every use of the function causes an iteration over the values to find the relevant command class. If it will cause problems in the future (currently I didn't see it with showing all 66 devices), we may have to go the node property way ...

Reply:
> We have to keep an eye on the performance though, since every use of the function causes an iteration over the values to find the relevant command class. If it will cause problems in the future (currently I didn't see it with showing all 66 devices), we may have to go the node property way

Sure! 👍🏼